spaceship/lib/spaceship/tunes/app_version.rb

```diff
@@ -215,6 +219,14 @@ def create_languages(languages)
         nil
       end
 
+      def current_build_number
+        if self.is_live?
+          build_version
+        else
+          candidate_builds.first.build_version
```

Comment:
no it will not fail - it will return `nil` - tested it, is that ok? - i am not sure what else would fit as a return value?

```
[30] pry(main)> app = Spaceship::Tunes::Application.find("krone.at-krone.at")
=> <Spaceship::Tunes::Application 
	apple_id="633492143", 
	name="Krone", 
	vendor_id="krone.at-krone.at", 
	bundle_id="krone.at-krone.at", 
	last_modified=1477673411000, 
	issues_count=0, 
	app_icon_preview_url="https://is3-ssl.mzstatic.com/image/thumb/Purple71/v4/90/a2/5e/90a25ecc-b333-6827-094d-1927a1404da8/mzl.hzffjpxt.png/340x340bb-80.png">
[31] pry(main)> version = app.edit_version.build_version
=> nil
[32] pry(main)> 
```